libbeat/kibana/client_config.go

```diff
@@ -6,7 +6,8 @@ import (
 	"github.com/elastic/beats/libbeat/common/transport/tlscommon"
 )
 
-type kibanaConfig struct {
+// ClientConfig to connect to Kibana
+type ClientConfig struct {
```

Comment:
Is there an other config then clientConfig for Kibana? Otherwise you could just call it `config`.